src/test/unit/scheduler/test_analysis.py

```diff
@@ -118,14 +117,14 @@ def test_skip_analysis_because_whitelist(self):
 class TestAnalysisSchedulerBlacklist:
 
     test_plugin = 'test_plugin'
-    fo = MockFileObject()
+    file_object = MockFileObject()
 
     class PluginMock:
         def __init__(self, blacklist=None, whitelist=None):
             if blacklist:
-                self.MIME_BLACKLIST = blacklist
+                self.MIME_BLACKLIST = blacklist  # pylint: disable=invalid-name
```

Comment:
better add `disable=invalid-name` to the top instead of adding it to each individual line